awsauthenticationlib/awsauthenticationlib.py

```diff
@@ -292,7 +298,13 @@ def _get_response(self, url, extra_cookies=None, params=None):
         self.logger.debug('Getting url :%s', url)
         response = requests.get(**arguments)
         if not response.ok:
-            raise ValueError('Response received: %s' % response.text)
+            if all([response.status_code == 400,  # pylint: disable= no-else-raise
```

Comment:
Woudn't it make more sense to parse the error message only once if the response is not ok and read the contents on your check?